src/main/java/uk/gov/hmcts/ccd/domain/model/std/AuditEvent.java

```diff
@@ -10,7 +10,7 @@
 import java.util.Map;
 
 public class AuditEvent extends Event {
-    @JsonProperty("id")
```

Comment:
Can we @JsonIgnore as we never want to return the actual physical db id of this object?